### DISCUSSION TITLE
copilot plugin for dataspell/pycharm is not working at the ipynb notebook.

### DISCUSSION THREAD

**Discussion by sdasd12312wasdasd** [community/community#32279] [Copilot Conversations]
copilot plugin for dataspell/pycharm is not working at the ipynb notebook.

**Comment by CasCard** [comment]
I have been facing the issue with VS Code ipynb as well

**Reply by davecheney** [reply]
@CasCard i'm sorry you are having difficulties. Looking at your telemetry it looks like you are making and receiving completions. Can you please tell me a bit more about what you tried, and what happened. Thank you.

**Comment by CasCard** [comment]
![image](https://user-images.githubusercontent.com/13176032/189086787-ed2b64ce-c8ef-4bf8-ba0c-11071c98097b.png)

Not getting suggestions even though I have logged in to Github as well as using the latest version of Pycharm

**Reply by davecheney** [reply]
@CasCard you said earlier that copilot wasn’t working for you in vscode. The screenshot above looks like IntelliJ. Which editors are you using?

**Reply by CasCard** [reply]
now VS Code seems working but Pycharm ipynb still not working

**Comment by davecheney** [comment]
Thank you. The first thing I’d suggest is triple check that you’re running the latest copilot plug-in. We update the plug-in often so you should try to keep up to date. 

Secondly, and I apologise that I’m not precise, I’m not an IntelliJ user, but is there some sort of debug or output log? Similar to the output window that vscode offers. If you can find that and paste the relevant parts that would be very helpful.

**Reply by shacharhelmer** [reply]
Hi @davecheney, perhaps the log provided in this report could help?
https://youtrack.jetbrains.com/issue/DS-3756
